src/crypto/backup.ts

```diff
@@ -118,10 +118,16 @@ export class BackupManager {
     public checkedForBackup: boolean; // Have we checked the server for a backup we can use?
     private sendingBackups: boolean; // Are we currently sending backups?
     private sessionLastCheckAttemptedTime: Record<string, number> = {}; // When did we last try to check the server for a given session id?
+    private clientRunning: boolean;
```

Comment:
The changes in this file do not match the PR title

Reply:
The test was failing, because something was attempting to log after the test end. It appeared that the backup manager is not shutdown when the client is stopped. So this was added to make the test work.

Reply:
I think these changes are required to get the tests to exit correctly. That said, ideally they could be a separate PR (or at least a separate commit).

Reply:
Its a functional change which should be in the changelo

Reply:
yeah. Which I think points towards making it a separate PR with a `T-Enhancement` label.